eahub/base/static/components/mailchimp/mailchimp.js

```diff
@@ -1,17 +0,0 @@
-(function ($) {
```

Comment:
What was the mailchimp code doing and why are we removing it?

Reply:
I have no idea really, we've quite a bit of dead code around, afaik this is one of those pieces.